sunpy/net/hek/hek.py

```diff
@@ -139,29 +139,6 @@ def _can_handle_query(cls, *query):
         return cls.check_attr_types_in_query(qr, required, optional)
 
 
-class HEKResponse(BaseQueryResponseTable):
```

Comment:
This needs deprecating instead of removing, just in case someone out there is using it

Reply:
This class was only added since 2.0, and this removal was the original genesis of this PR, so it's actually avoiding a breaking change rather than adding one :rofl: